accepted/2020/feature-switch.md

```diff
@@ -77,11 +77,60 @@ Ultimately some of these switches may also need VS UI integration, but for now m
 
 The ability to specify these from MSBuild projects as properties also means that different templates and SDKs can set different defaults - so for example Blazor or Xamarin SDKs may choose to turn off some of these features by default.
 
+Example definition of a feature flag with SDK support in [`Microsoft.NET.Sdk.targets`](https://github.com/dotnet/sdk/blob/36ef8b2aa8e5d579c921704bdab69a7407936889/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets) or by SDK components:
+```xml
+<ItemGroup>
+    ...
+    <RuntimeHostConfigurationOption Include="System.Runtime.OptionalBehavior"
+                                    Condition="'$(OptionalBehavior)' != ''"
+                                    Value="$(OptionalBehavior)" />
+    ...
+</ItemGroup>
+```
+
+The name of the property should be picked so that it's clear what `true`/`false` mean. In this case, the idea is that `false` means the feature will be disabled. Other cases (for example `InvariantGlobalization`) might have the opposite polarity.
+
 ### Generate the right input for the linker in SDK
 
-The names/values from `RuntimeHostConfigurationOptions` will be passed to the ILLink task, which will apply any feature substitutions defined in `substitutions.xml`. The substitutions file format will be extended to condition the substitutions based on the feature name/value.
+All names/values from `RuntimeHostConfigurationOptions` will be passed to the ILLink task, which will apply any feature substitutions defined in [`substitutions.xml`](https://github.com/mono/linker/blob/master/src/linker/README.md#using-custom-substitutions). The substitutions file format will be extended to condition the substitutions based on the feature name/value. Any `RuntimeHostConfigurationOptions` which do not have feature implementations in `substitutions.xml` will not result in any modifications to the IL.
 
-The assembly defining a feature will carry an embedded `substitutions.xml` that defines the implementation of the corresponding IL property when the feature switch is set.
+Example of a feature implementation:
+
+```xml
+<linker feature="System.Runtime.OptionalBehavior" value="false">
```

Comment:
This will need some tweaking as we'll need to support more than one feature per `substitution.xml`.
I assume the 'value' acts as sort of a condition - "If System.Runtime.OptionalBehavior==false then do this".

Reply:
I moved it to the `assembly` level in this example. I implemented it like you suggested in https://github.com/mono/linker/pull/1132/files - see if that looks ok to you.